cdk/README.md

```diff
@@ -286,138 +412,94 @@ var gossipKeySecretArn= process.env.CONSUL_GOSSIP_KEY_ARN || `$CONSUL_GOSSIP_KEY
         retryJoin,
-        consulClientSecurityGroup: envProps.clientSecurityGroup,
-        consulServerSecurityGroup: envProps.serverSecurityGroup,
+        consulClientSecurityGroup: consulClientSecurityGroup,
+        consulServerSecurityGroup: consulServerSecurityGroup,
         consulCACert: serverProps.agentCASecret,
         gossipEncryptKey: serverProps.gossipKeySecret,
         tls: true,
         consulDatacenter: serverProps.serverDataCenter,
       };
-    }
-  }
-}
-```
-
-### Modify app entry point `bin/app.ts`
-```ts
-// Microservices with Consul Client
-const microservices = new Microservices(app, 'ConsulMicroservices', environment.props, serverProps);
-```
 
-### Add the `name` service in `lib/microservices.ts`
-```ts
+      // NAME service
       const nameDescription = new ecs_extensions.ServiceDescription();
       nameDescription.add(new ecs_extensions.Container({
-        cpu: 1024,
-        memoryMiB: 2048,
+        cpu: 512,
+        memoryMiB: 1024,
         trafficPort: 3000,
         image: ecs.ContainerImage.fromRegistry('nathanpeck/name')
       }));
       nameDescription.add(new consul_ecs.ECSConsulMeshExtension({
         ...baseProps,
+        port: 3000,
         serviceDiscoveryName: 'name',
       }));
       nameDescription.add(new ecs_extensions.AssignPublicIpExtension());
       const name = new ecs_extensions.Service(this, 'name', {
         environment: envProps.ecsEnvironment,
         serviceDescription: nameDescription
       });
-```
 
-### Add the `greeting` service in `lib/microservices.ts`
-```ts
       // GREETING service
       const greetingDescription = new ecs_extensions.ServiceDescription();
       greetingDescription.add(new ecs_extensions.Container({
-        cpu: 1024,
-        memoryMiB: 2048,
+        cpu: 512,
+        memoryMiB: 1024,
         trafficPort: 3000,
         image: ecs.ContainerImage.fromRegistry('nathanpeck/greeting')
       }));
       greetingDescription.add(new consul_ecs.ECSConsulMeshExtension({
         ...baseProps,
+        port: 3000,
         serviceDiscoveryName: 'greeting',
       }));
       greetingDescription.add(new ecs_extensions.AssignPublicIpExtension());
       const greeting = new ecs_extensions.Service(this, 'greeting', {
         environment: envProps.ecsEnvironment,
         serviceDescription: greetingDescription,
       });
-```
 
-### Add the `greeter` service in `lib/microservices.ts`
-```ts
       // GREETER service
       const greeterDescription = new ecs_extensions.ServiceDescription();
       greeterDescription.add(new ecs_extensions.Container({
-        cpu: 1024,
-        memoryMiB: 2048,
+        cpu: 512,
+        memoryMiB: 1024,
         trafficPort: 3000,
-        image: ecs.ContainerImage.fromRegistry('nathanpeck/greeter'),
+        image: ecs.ContainerImage.fromRegistry('nathanpeck/greeter')
       }));
       greeterDescription.add(new consul_ecs.ECSConsulMeshExtension({
         ...baseProps,
+        port: 3000,
```

Comment:
same as above

Reply:
addressed in [here](https://github.com/aws-samples/amazon-ecs-fargate-consul-connect-example/pull/16/commits/a8f0b8f5a6c7e8acd1e530034a89534277ebf6f6)

Reply:
README [fix](https://github.com/aws-samples/amazon-ecs-fargate-consul-connect-example/pull/16/commits/c43e473fc4a3a7b2b8f0158e7fbe7c9e7e7506c0#diff-b0b0aa5ef277f2eeabb6b5f834c43d21584c4deaa2f5fe02748cf2dd1a3025d0L455)